L/libjulia/common.jl

```diff
@@ -234,10 +249,9 @@ function build_julia(ARGS, version)
     # Dependencies that must be installed before this package can be built/used
     dependencies = [
         Dependency("LibUnwind_jll"),
-        Dependency(PackageSpec(name="PCRE2_jll", version=v"10.31")),
+        Dependency(PackageSpec(name="PCRE2_jll"), compat="10.31"),
```

Comment:
We might as well stop using PackageSpec everywhere in this file... thoughts on that?
```suggestion
        Dependency("PCRE2_jll", compat="10.31"),
```